boards/tenstorrent/tt_blackhole/bootfs/p150a-bootfs.yaml

```diff
@@ -41,6 +41,7 @@ images:
 
   # Board Mgmt FW
   - name: bmfw
+    padto: 0xb000
```

Comment:
these we probably don't need. It should be the default to align up to a block erase boundary